java/test/jmri/jmrit/logix/BlockOrderTest.java

```diff
@@ -1,30 +1,91 @@
 package jmri.jmrit.logix;
 
+import jmri.InstanceManager;
+import jmri.SignalHead;
+import junit.framework.Test;
+import junit.framework.TestCase;
+import junit.framework.TestSuite;
 import org.junit.After;
 import org.junit.Assert;
 import org.junit.Before;
-import org.junit.Ignore;
-import org.junit.Test;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**
  *
  * @author Paul Bender Copyright (C) 2017	
  */
-public class BlockOrderTest {
```

Comment:
I'm with @rhwood  on this.  The tests need to remain in JUnit4 format. 

I consider changing a test from JUnit4 to JUnit3 to be a subtractive change.